netbout-web/src/main/java/com/netbout/rest/TkAppAuth.java

```diff
@@ -89,54 +89,65 @@
      * @param take Take
      * @param pass Last Pass on Chain
      * @return Authenticated take
+     * @todo #685:30m/DEV Need to implement PsBasic.Entry.
+     *  PsFake usage should be changed to
+     *  PsBasic(
+     *  Manifests.read("Netbout-Basic-Realm"),
+     *  new PsBasic.Default(
+     *  Manifests.read("Netbout-Basic-User"),
+     *  Manifests.read("Netbout-Basic-Pwd")))
+     *  But PsBasic.Default like class is currently missing. See takes #349.
      */
     private static Take make(final Take take, final Pass pass) {
         return new TkAuth(
             take,
-            new PsChain(
-                new PsByFlag(
-                    new PsByFlag.Pair(
-                        PsGithub.class.getSimpleName(),
-                        new PsGithub(
-                            Manifests.read("Netbout-GithubId"),
-                            Manifests.read("Netbout-GithubSecret")
-                        )
-                    ),
-                    new PsByFlag.Pair(
-                        PsFacebook.class.getSimpleName(),
-                        new PsFacebook(
-                            Manifests.read("Netbout-FbId"),
-                            Manifests.read("Netbout-FbSecret")
-                        )
-                    ),
-                    new PsByFlag.Pair(
-                        PsGoogle.class.getSimpleName(),
-                        new PsGoogle(
-                            Manifests.read("Netbout-GoogleId"),
-                            Manifests.read("Netbout-GoogleSecret"),
-                            "http://www.netbout.com/?PsByFlag=PsGoogle"
+            new PsTwice(
+                new PsFake(true),
```

Comment:
@dmzaytsev Originally, I did it just like your proposal, but that lead to massive test failure. So, this would involve reworking of the mocks for the test. I'd like to leave this task to the one who will be fixing `@todo #685:30m/DEV` integrating actual `PsBasic`, I find it fair enough. wdyt?